src/shared/codelens/localLambdaRunner.ts

```diff
@@ -499,14 +499,20 @@ export const invokeLambdaFunction = async (params: {
         templatePath: params.samTemplatePath,
         eventPath,
         environmentVariablePath,
-        debugPort: (params.isDebug) ? params.debugConfig.port.toString() : undefined,
+        debugPort: (params.isDebug) ?
+            (params.debugConfig ? params.debugConfig.port.toString() : undefined) : undefined,
```

Comment:
This feels dirty and formatted wrong

Reply:
Agreed. `params.isDebug` doesn't seem to have a purpose--it's redundant with `!!port` and requires us to validate that `isDebug` and `port` are consistent with each other. Let's keep the messy version for now, and remove `isDebug` in a separate PR.

For formatting, consider something like:

```typescript
(params.isDebug && params.debugConfig) ?
    params.debugPort.toString() :
    undefined
```